docusaurus/docs/dev-docs/plugins/development/plugin-cli.md

```diff
@@ -3,95 +3,92 @@ title: Plugin CLI
 description: Reference documentation for Strapi's Plugin CLI commands
 displayed_sidebar: devDocsSidebar
 tags:
-- backend server
-- plugin CLI
-- plugins
-- plugins development
-- strapi plugin
+  - backend server
+  - plugin CLI
+  - plugins
+  - plugins development
+  - strapi plugin
 ---
 
-import NotV5 from '/docs/snippets/_not-updated-to-v5.md'
+import NotV5 from '/docs/snippets/\_not-updated-to-v5.md'
 
 # Plugin CLI reference
 
 <NotV5/>
 
-:::caution
-The Plugin CLI is currently experimental.
-:::
-
-The Plugin CLI is set of commands orientated around developing plugins to use them as local plugins or to publish them on NPM and/or submit them to the Marketplace.
+The Plugin CLI (@strapi/sdk-plugin) is set of commands orientated around developing plugins to use them as local plugins or to publish them on NPM and/or submit them to the Marketplace.
 
+The commands are in their own repository called
```

Comment:
missing end of sentence? 🤔

Reply:
Haha, yes, I wrote it before we had settled the question. I'll add it.